exec/java-exec/src/test/java/org/apache/drill/exec/store/log/TestLogReader.java

```diff
@@ -765,4 +777,16 @@ public void testFirewallSchema() throws RpcException {
     RowSetUtilities.verify(expected, result);
     result.clear();
   }
-}
+
+  @Test
+  public void testIssue7853() throws Exception {
+    thrownException.expect(UserRemoteException.class);
+    thrownException.expectMessage("is not valid for type TIMESTAMP");
+    String sql = "SELECT * FROM cp.`regex/issue7853.log3`";
+    QueryBuilder builder = client.queryBuilder().sql(sql);
+    RowSet sets = builder.rowSet();
+
+    assertEquals(2, sets.rowCount());
+    sets.clear();
```

Comment:
This is fine. There is a simpler solution:

```
   QuerySummary result = client.queryBuilder().sql(sql).run();
   assertEquals(2, result.records);
```

However, all this test does is assert not crash. What you really want to know is that the format worked. So, we want to build a result set with expected results as done in other tests. To avoid having to crate a zillion fields, change your query to select only the fields of interest.